frontend/src/libs/packages/store/store.package.ts

```diff
@@ -23,6 +26,7 @@ type ExtraArguments = {
   authApi: typeof authApi;
   userApi: typeof userApi;
   storage: typeof storage;
+  notification: Notification;
```

Comment:
We don't need to export Notification
```suggestion
  notification: typeof notification;
```